it/main/PostgresTest.java

```diff
@@ -61,13 +64,15 @@ public EntityManagerFactory getEMFForProductionLikeDatabase() {
 
     public EntityManagerFactory getEMFForEmptyTestDatabase() {
         Map<String, String> databaseParameters = new HashMap<String, String>() {{
-            put(JDBC_URL, "jdbc:h2:mem:test");
-            put(JDBC_USER, "SA");
-            put(JDBC_PASSWORD, "");
-            put(JDBC_DRIVER, "org.h2.Driver");
-            put(DDL_GENERATION, DROP_AND_CREATE);
-            put(DDL_GENERATION_MODE, DDL_DATABASE_GENERATION);
-            put(TARGET_DATABASE, H2Platform.class.getName());
+            put(JPA_JDBC_URL, "jdbc:h2:mem:test");
+            put(JPA_JDBC_USER, "SA");
+            put(JPA_JDBC_PASSWORD, "");
+            put(JPA_JDBC_DRIVER, "org.h2.Driver");
+            put(HBM2DDL_AUTO, "create-drop");
+            put(DIALECT, H2Dialect.class.getName());
+//            put(DDL_GENERATION, DROP_AND_CREATE);
```

Comment:
I imagine this is commented out so that in future tests we can drop and create if needed (even though from what I see, it's an in memory DB so we don't even need this to begin with?)